docs/essentials/overriding.md

```diff
@@ -0,0 +1,108 @@
+# Overriding
+
+Overriding in Storefront-x enables you to change visual or functional behaviour of one module with another module, without modifing original files. Imagine, you want to change the default template of the component or change the default behaviour of the service in some common-module, overriding is perfect match for you.
```

Comment:
Overriding alloes you to easily replace parts of SFX and thus modifying/adding/removing functionality. For example you can easily override how adding to cart works, how header/footer looks, how catalog is displayed etc. 

Overriding works is file based so basically any file (page, component, service, composable, ...) can be overridden.